### PR TITLE
Fix some cgroups test failures

### PR DESCRIPTION
**Pull request by vstumpf**
#### Bug/feature Description
* Some tests were failing.

#### Affected Platform(s)
* Centos 7

#### Cause / Analysis / Design
* The vnodes were being created before the hook was imported, leading to some timing problems.
* The argument to wait_and_read_file had the wrong name.
* The test was failing on systems with memsw disabled.

#### Solution Description
* Switch the order of vnode creation/hook import.
* Change the name of the argument
* Copy the functional test's flexibility.

#### Testing logs/output
* [cg-20180530224328.log](https://github.com/PBSPro/pbspro/files/2055945/cg-20180530224328.log)
* Updated: [cg-20180531213853.log](https://github.com/PBSPro/pbspro/files/2060258/cg-20180531213853.log)

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [x] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to this pull request** as evidence of testing/verification.

### CODE REVIEW

File: test/tests/performance/pbs_cgroups_stress.py [hunk 1]
```diff
@@ -84,7 +96,7 @@ def setUp(self):
         },
         "memsw":
         {
-            "enabled"         : true,
+            "enabled"         : %s,
```
[the remark this answers is not in the document]
Comment (vstumpf) [reply test/tests/performance/pbs_cgroups_stress.py]: done

**Comment by mike0042** [comment]
Please rebase in preparation for merge.

**Comment by vstumpf** [comment]
Removed some unnecessary return code checks for `self.server.manager`
Simplified some code in `load_hook`
Fixed an issue where a directory could not be deleted if a test exited early.

**Comment by mike0042** [comment]
Still looks good.

**Comment by bhroam** [comment]
I merged in the PR

**Pull request merged**